.circleci/config.yml

```
@@ -52,14 +52,14 @@ jobs:
             tasksExists=$(aws --output text ecs list-task-definitions --family-prefix ${awsFamily})
             echo "existsTaskDefinition? --${tasksExists}--"
             if [ ! -z "$tasksExists" ]; then
-              isTaskRunning=$(aws --output text ecs list-tasks --cluster caleydo --family ${awsFamily})
+              isTaskRunning=$(aws --output text ecs list-tasks --cluster JKU_ASG_Cluster --family ${awsFamily})
```

I think, the production cluster name is `ordino-public`. 

```suggestion
              isTaskRunning=$(aws --output text ecs list-tasks --cluster ordino-public --family ${awsFamily})
```